spring-cloud-gcp-data-spanner/src/test/java/com/google/cloud/spring/data/spanner/SimpleSpannerRepositoryTests.java

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
There is a SpannerRepositoryImplTests in the repo that already has some test on `SimpleSpannerRepository`, so deleted this and added unit test there instead.